python/BioSimSpace/Process/_openmm.py

```diff
@@ -68,6 +67,34 @@ class OpenMM(_process.Process):
     # Dictionary of platforms and their OpenMM keyword.
     _platforms = {"CPU": "CPU", "CUDA": "CUDA", "OPENCL": "OpenCL"}
 
+    # Special cases for generate config when using ATM protocols.
+    def __new__(
+        cls,
+        system=None,
+        protocol=None,
+        reference_system=None,
+        exe=None,
+        name="openmm",
+        platform="CPU",
+        work_dir=None,
+        seed=None,
+        property_map={},
+        **kwargs,
+    ):
+        from ._atm import OpenMMAToM
+
+        # would like to use issubclass but _Protocol._AToM is not exposed
+        if (
+            isinstance(protocol, _Protocol.AToMMinimisation)
+            or isinstance(protocol, _Protocol.AToMEquilibration)
+            or isinstance(protocol, _Protocol.AToMAnnealing)
+            or isinstance(protocol, _Protocol.AToMProduction)
+        ):
```

Comment:
While the base `_AToM` protocol isn't exposed via the public API, you should just be able to import and use it directly, e.g.:

```python
from ..Protocol._atm import _AToM

if isinstance(protocol, _AToM):
    ...
```

Note that `isinstance` will work because this is an actual instance, rather than a class.

Reply:
This has been updated accordingly (https://github.com/OpenBioSim/biosimspace/blob/feature_AToM/python/BioSimSpace/Process/_openmm.py#L88)